Source/Plugin.BLE.iOS/Device.cs

```diff
@@ -135,5 +135,11 @@ public void Update(CBPeripheral nativeDevice)
             Trace.Message($"Request MTU is not supported on iOS.");
             return await Task.FromResult((int)_nativeDevice.GetMaximumWriteValueLength(CBCharacteristicWriteType.WithoutResponse));
         }
+
+        protected override async Task<bool> UpdateConnectionIntervalNativeAsync(ConnectionInterval interval)
+        {
+            Trace.Message("Cannot update connection inteval on iOS. Returning TRUE ..");
+            return await Task.FromResult(true);
```

Comment:
According to async: I will change it to "return true".

About the return value:
Hm I was not sure about the expected behavior from the users perspective.... maybe "false" would be better since its not possible to apply anything; I agree!